tests/Export/SalesChannelServiceTest.php

```diff
@@ -43,11 +45,32 @@ public function testNothingConfiguredForGivenShopkeyReturnsNull(): void
     private function getSalesChannelService(): SalesChannelService
     {
         /** @var EntityRepository $configRepository */
-        $configRepository = $this->getContainer()->get('system_config.repository');
+        $configRepository = $this->getContainer()->get('finsearch_config.repository');
 
         return new SalesChannelService(
             $configRepository,
             $this->getContainer()->get(SalesChannelContextFactory::class)
         );
     }
+
+    public function testCorrectLanguageIdIsConfiguredInSalesChannelContext(): void
+    {
+        $languageId = Uuid::randomHex();
+        $this->createLanguage($languageId);
+
+        $configShopkey = '12341234123412341234123412341234';
+
+        $salesChannelService = $this->getSalesChannelService();
+        $currentSalesChannel = $this->buildSalesChannelContext(
+            Defaults::SALES_CHANNEL,
+            'http://test.de',
+            null,
+            $languageId
+        );
+
+        $this->enableFindologicPlugin($this->getContainer(), $configShopkey, $currentSalesChannel);
+        $salesChannel = $salesChannelService->getSalesChannelContext($currentSalesChannel, $configShopkey);
```

Comment:
Add `null` as first parameter. Otherwise this test isn't really testing anything.

Reply:
@TheKeymaster We need the context to search the repository for the shopkey. and to get the token for creating a sales channel context.

Reply:
Ok.